docs/applications/project-management/create_a_private_python_package_repository.md

```diff
@@ -200,15 +201,15 @@ The next step is setting up a server to host a package index. This guide will us
     The repository should be accessible through `192.0.2.0` by default on port 80 where `192.0.2.0` is the public of the Linode.
 
 # Download From a Client
-Recall the rather long flags declared with `pip` in order to download from a different specified repository. Create a configuration file containing the IP of your public server will simplify usage.
+Recall the rather long flags declared with `pip` in order to download from a specified repository. Creating a configuration file containing the IP of your public server will simplify usage.
 
 1.  On a client computer, create a `.pip` directory in the home directory. Inside the this directory, create `pip.conf` with the following:
 
     {:.file}
     pip.conf
     :   ~~~
     [global]
-    extra-index-url = http://192.0.2.0
+    extra-index-url = http://192.0.2.0:8080/
```

Comment:
This should in theory work on port 80. If it's also listening on port 8080, that means an instance of Bottle is still running